### PR TITLE
Placeholder cycle 

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

* adds a cycle of placeholders to guide the user more on what they can search for

### CODE REVIEW

**Comment by nelsonic** [comment]
👀

File: lib/search-bar/index.js [hunk 1]
```diff
@@ -68,7 +70,19 @@ export default class SearchBarContainer extends Component {
...
+    this.setState({
+      placeholder: placeholders[counter]
+    });
+    counter++;
+    if (counter === placeholders.length) {
+      counter = 0;
+    }
+  }
+  componentDidMount () {
+    setInterval(this.placeholderSwitch.bind(this), 2000);
```
Comment (nelsonic) [inline lib/search-bar/index.js]: will this over-write text input by the _person_ while they are typing? 
(_I don't think it will... but how do we check?_)

**Comment by jackcarlisle** [comment]
@nelsonic I'll create an issue now to open the discussion. I'll add tests for the functionality too

File: lib/search-bar/index.js (hunk 1 above)
Comment (nelsonic) [inline lib/search-bar/index.js]: Confirmed the browser behaviour: https://jsfiddle.net/eta8pm9k/1/

**Comment by jackcarlisle** [comment]
https://jsfiddle.net/eta8pm9k/1/

**Comment by nelsonic** [comment]
Merging. 👍

**Pull request merged**